client-v2/src/constants/theme.ts

```diff
@@ -42,7 +42,9 @@ const capiInterface = {
 const front = {
   frontListBorder: '#5E5E5E',
   frontListLabel: shared.colors.greyMediumLight,
-  frontListButton: shared.colors.greyDark
+  frontListButton: shared.colors.greyDark,
+  minWidth: 380,
```

Comment:
These values were attached to individual components. I've moved them into our theme.